src/main/java/com/arpnetworking/metrics/impl/BaseFileSink.java

```diff
@@ -414,5 +453,7 @@ protected void validate(final List<String> failures) {
         private static final Boolean DEFAULT_PRUDENT = Boolean.FALSE;
         private static final Boolean DEFAULT_DROP_WHEN_QUEUE_FULL = Boolean.FALSE;
         private static final Integer DEFAULT_MAX_QUEUE_SIZE = Integer.valueOf(500);
+        private static final String DEFAULT_MAX_FILE_SIZE = "100MB";
+        private static final String DEFAULT_TOTAL_SIZE_CAP = "20GB";
```

Comment:
The relevant logic lives in the ArchiveRemover which is common to both rolling policies.  MaxSize will limit total size under the current logback release as long as you don't hit that limit before 2 rolling periods.  (2 hours for most of our services)  This does make writing a unit test a bit of a pain.

Max history does equal the number of files when doing only time based rolling but that does not currently remain true when you start rolling by size.